rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java

```diff
@@ -74,6 +75,7 @@ void setUpOnEachTest() {
         activationsBeforeForks = ActivationConfigsForTest.genesis().forBlock(0);
         activationsAfterForks = ActivationConfigsForTest.all().forBlock(0);
         bridgeSupportBuilder = new BridgeSupportBuilder();
+        whitelistSupport = mock(WhitelistSupport.class);
```

Comment:
i think there's no need to declare this as a global variable, is being used in a few tests

Reply:
It is being used in 3 methods excluding the `setup` method. This is a good way to avoid repeating code and get a higher cohesion in the class.